### DISCUSSION TITLE
Docs incorrectly state filtering on severity accepts more than a single value

### DISCUSSION THREAD

**Discussion by axi0m** [aquasecurity/trivy#7471] [Documentation]
### Description

I was reviewing this https://aquasecurity.github.io/trivy/v0.55/tutorials/misconfiguration/terraform/ and it says:

> If you are presented with lots and lots of misconfiguration across different files, you might want to filter or the misconfiguration with the highest severity:
`trivy config --severity CRITICAL, MEDIUM terraform-infra `

However, that does not work in my experience with trivy `Version: 0.54.1`:

`trivy config --severity CRITICAL, MEDIUM, HIGH main.tf`

`2024-09-09T10:30:07-05:00	FATAL	Fatal error	multiple targets cannot be specified`

If I pass a single severity rating to the `--severity` parameter it runs successfully. 

`trivy config --severity CRITICAL main.tf`

`2024-09-09T10:30:36-05:00	INFO	[misconfig] Misconfiguration scanning is enabled`

### Link

https://aquasecurity.github.io/trivy/v0.55/tutorials/misconfiguration/terraform/

### Suggestions

You're using Material UI for MkDocs, kindly [add an admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) that says:

"You can only pass a single severity value, multiple targets are not supported."

**Comment by simar7** [comment]
Values are comma separated not comma-space. Correct invocation would be the following:

`trivy config --severity CRITICAL,MEDIUM terraform-infra`

Note the lack of space between the severities.

**Reply by axi0m** [reply]
That works! Thank you - Do the docs need to be updated to reflect that?

**Reply by simar7** [reply]
I think it is documented well IMO

```
 trivy config -h | grep severities
  -s, --severity strings           severities of security issues to be displayed (UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL) (default [UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL])
```
